frontend/src/pages/RecipientRecord/pages/PrintGoals.js

```diff
@@ -25,7 +26,7 @@ export default function PrintGoals({ location, recipientId, regionId }) {
           regionId,
           sortConfig.sortBy,
           sortConfig.direction,
-          sortConfig.offset,
+          OFFSET,
```

Comment:
This always being zero is intentional?

Reply:
yeah, this function will be outta here when you are able to select specific goals